util/topo/model.go

```diff
@@ -0,0 +1,176 @@
+// Copyright 2022 PingCAP, Inc. Licensed under Apache-2.0.
+
+package topo
+
+type CompStatus string
+
+const (
+	CompStatusUnknown     CompStatus = "unknown"
+	CompStatusUnreachable CompStatus = "unreachable"
+	CompStatusUp          CompStatus = "up"
+	CompStatusTombstone   CompStatus = "tombstone"
+	CompStatusLeaving     CompStatus = "leaving"
+	CompStatusDown        CompStatus = "down"
+)
+
+type Kind string
+
+const (
+	KindTiDB         Kind = "tidb"
+	KindTiKV         Kind = "tikv"
+	KindPD           Kind = "pd"
+	KindTiFlash      Kind = "tiflash"
+	KindAlertManager Kind = "alert_manager"
+	KindGrafana      Kind = "grafana"
+	KindPrometheus   Kind = "prometheus"
+)
+
+type PDInfo struct {
+	GitHash        string
+	Version        string
+	IP             string
+	Port           uint
+	DeployPath     string
+	Status         CompStatus
+	StartTimestamp int64 // Ts = 0 means unknown
+}
+
+var _ Info = &PDInfo{}
```

Comment:
This is used to claim that PDInfo must implement Info interface, right?

Reply:
Yes.